type/__apt_source/files/source.list.template

```diff
@@ -2,13 +2,14 @@
 set -u
 
 entry="$uri $distribution $component"
+
 cat << DONE
 # Created by cdist ${__type##*/}
 # Do not change. Changes will be overwritten.
 #
 
 # $name
-deb ${forcedarch} $entry
+deb ${options} $entry
```

Comment:
I propose to use:
`deb ${options:+${options} }${entry}`

So that we don't get a double-space when no options are used.